locations/spiders/bar_burrito_ca.py

```diff
@@ -0,0 +1,20 @@
+import html
+
+from locations.items import Feature
+from locations.storefinders.wp_store_locator import WPStoreLocatorSpider
+
+
+class BarBurritoCASpider(WPStoreLocatorSpider):
+    name = "bar_burrito_ca"
+    item_attributes = {"brand": "BarBurrito", "brand_wikidata": "Q123765650"}
```

Comment:
When https://www.wikidata.org/wiki/Q104844862 works

```suggestion
    item_attributes = {"brand": "BarBurrito", "brand_wikidata": "Q104844862"}
```